build.gradle

```diff
@@ -313,6 +313,12 @@ project(':core') {
     from sourceSets.test.output
   }
 
+  task publishDocs(type: Jar) {
```

Comment:
I'm not sure JAR is the right way to publish user-facing docs. Most projects either generate tarball, throw the docs into target/site (build/site in our case) or both.